server/http_tunnel.go

```diff
@@ -21,11 +24,11 @@ import (
 
 // StartHTTPTunnel starts a server that accepts to the HTTP CONNECT method to proxy arbitrary TCP connections.
 // It can be used to tunnel HTTPS connections.
-func StartHTTPTunnel(addr, externalProxy string) error {
-	return StartHTTPSTunnel(addr, externalProxy, "", "", "", "")
+func StartHTTPTunnel(addr, externalProxy string, rootCAFile string) error {
```

Comment:
I know you need to pass parameter `rootCAFile` to `StartHTTPSTunnel()`. But it might be better to not pass in any tls related configuration for a function that start up non-tls HTTP tunnel.

Reply:
The HTTP tunnel also supports forwarding its traffic through another proxy.  The other proxy can be served over TLS, so the rootCAFile is needed for that.

Reply:
then it sounds good to me